codegen/external/README.md

```diff
@@ -6,3 +6,6 @@ repository, with some modifications to spirv.core.grammar.json. So there is
 a copy of it in this directory.
 
 [spirv-headers]: https://github.com/KhronosGroup/SPIRV-Headers
+
+Modifications are:
+  - "class" field
```

Comment:
Might as well be a bit more specific here, just in case we end up modifying enough that diffs get confusing if we have to update it.

Reply:
The thing is, my PR doesn't introduce the divergence, it's already there.
I just left this as a beginning of some kind of a doc to reflect that. I can remove this readme change if you think it would be nicer, I don't think it's harmful though.

Reply:
Fair enough, I didn't realise it had already been modified.

Reply:
Yeah, I introduced the modification to facilitate code autogen. Should really consider upstream this. It's a pain to maintain.